myuw/test/api/instructor_schedule.py

```diff
@@ -40,3 +40,30 @@ def test_bill_current_term(self):
             self.assertEqual(
                 len(data['sections'][1]['grade_submission_delegates']),
                 1)
+
+
+@require_url('myuw_instructor_schedule_api',
+             kwargs={'year': 2013, 'quarter': 'summer'},
+             message="Specific term instructor URLs not configured")
+class TestInstructorTermSchedule(MyuwApiTest):
+
+    def get_schedule(self, **kwargs):
+        return self.get_response_by_reverse(
+            'myuw_instructor_schedule_api',
+            kwargs=kwargs,)
+
+    def test_bill_future_term(self):
+        with self.settings(RESTCLIENTS_SWS_DAO_CLASS=FDAO_SWS,
+                           RESTCLIENTS_PWS_DAO_CLASS=FDAO_PWS,
+                           RESTCLIENTS_CANVAS_DAO_CLASS=FDAO_Canvas):
```

Comment:
Could you use the decorators defined in
https://github.com/uw-it-aca/uw-restclients/blob/master/restclients/test/__init__.py ?
Please update myuw.test.__init__.py and import from from estclients.test